demo-project/src/demo_project/pipelines/data_ingestion/nodes.py

```diff
@@ -116,3 +116,9 @@ def combine_shuttle_level_information(
     working_table = combined_table.dropna(how="any")
     id_columns = [x for x in working_table.columns if x.endswith("id")]
     return working_table, working_table[id_columns]
+
+
+def apply_basic_logic_to_sql_example_data(
+    sqlExampleParam
+) -> Tuple[pd.DataFrame, pd.DataFrame]:
+    return sqlExampleParam
```

Comment:
Is this still needed?

Reply:
Ah crap! Missed that earlier, not needed yeah

Reply:
Re UI, good call! On:
- **QA Notes:** Added now
- **Testing UI on .js side:** Shouldn't be necessary; front-end side does not seem to distinguish between 'data' and 'task' nodes - it just treats them all as one type (node). 'Preview' is only enabled if back-end feeds it a 'code' value; this PR hijacks the backend to feed the same thing for its data nodes. So for front-end, the current UI testing it does for its 'preview' nodes will cover both task and data nodes from the get-go